spec/spec_helper_acceptance.rb

```diff
@@ -1,35 +1,26 @@
-require 'beaker-rspec'
+require 'beaker-rspec/spec_helper'
+require 'beaker-rspec/helpers/serverspec'
 
-config = {
-  'main' => {
-    'logdir' => '/var/log/puppet',
-    'vardir' => '/var/lib/puppet',
-    'ssldir' => '/var/lib/puppet/ssl',
-    'rundir' => '/var/run/puppet',
-  },
-}
-
-# Install latest puppet from puppetlabs.com
 hosts.each do |host|
-  on hosts, install_puppet
+  # Install Puppet
+  install_puppet
 end
-# Explicitly configure puppet to avoid warnings
-configure_puppet(config)
 
 RSpec.configure do |c|
-  module_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
+  # Project root
+  proj_root = File.expand_path(File.join(File.dirname(__FILE__), '..'))
 
+  # Readable test descriptions
   c.formatter = :documentation
 
+  # Configure all nodes in nodeset
   c.before :suite do
-    on 'debian', 'apt-get -y install apt-transport-https'
-
-    # Install this module for testing
-    puppet_module_install(:source => module_root, :module_name => 'telegraf')
-
+    # Install module
+    # We assume the module is in auto load layout
+    puppet_module_install(:source => proj_root, :module_name => "#{File.basename(proj_root)}")
+    # Install dependancies
     hosts.each do |host|
-      on host, puppet('module', 'install', 'puppetlabs-stdlib'), { :acceptable_exit_codes => [0,1] }
-      on host, puppet('module', 'install', 'puppetlabs-apt'), { :acceptable_exit_codes => [0,1] }
+      on host, puppet('module', 'install', 'puppetlabs-stdlib', '-v 4.3.2')
```

Comment:
For the Debian acceptance tests to work, we also need to install `puppetlabs-apt` and the `apt-transport-https` package.